components/ViewItem.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import { v4 as uuidv4 } from "uuid";
+import Link from "next/link";
+import { myViewsContext } from "@/pages/_app";
+import { useContext } from "react";
+
+export default function ViewItem() {
+  const globalViews = useContext(myViewsContext);
+
+  return (
+    <Ul>
+      {globalViews.map((view) => (
+        <Li key={uuidv4()}>
```

Comment:
`key`s should be stable ids. random generated keys are not ideal, since this breaks the key performace optimization of react.

Reply:
got it 👍